plugins/terraform/src/components/TerraformLatestRun/TerraformLatestRun.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect } from 'react';
+import { CircularProgress, Grid } from '@material-ui/core';
+import { ResponseErrorPanel } from '@backstage/core-components';
+import { TerraformLatestRunContent } from '../TerraformLatestRunContent';
+import { useRuns } from '../../hooks';
+import { Run } from '../../hooks/types';
+import {
+  isTerraformAvailable,
+  TERRAFORM_WORKSPACE_ANNOTATION,
+  TERRAFORM_WORKSPACE_ORGANIZATION,
+} from '../../annotations';
+import { MissingAnnotationEmptyState, useEntity } from '@backstage/plugin-catalog-react';
+
+
+export const TerraformLatestRun = () => {
+
+  const { entity } = useEntity();
+
+  const { annotations } = entity.metadata;
+  const organization = annotations?.[TERRAFORM_WORKSPACE_ORGANIZATION] ?? 'undefined';
+  const workspaceName = annotations?.[TERRAFORM_WORKSPACE_ANNOTATION] ?? 'undefined';
+
+  const { data, isLoading, error, refetch } = useRuns(organization, workspaceName);
+
+  useEffect(() => {
```

Comment:
This shouldn't be required here.

Reply:
This needs to be declared here as it calls `refetch` on loading the page.  Without this we get no data.